scripts/k8s-storage/driver-rbd-rwo.yaml

```diff
@@ -0,0 +1,72 @@
+---
+ShortName: cephcsi-rbd-rwo-test
+StorageClass:
+  FromExistingClassName: k8s-storage-e2e-rbd-rwo
+  # FromFile: sc-rbd.yaml
+
+SnapshotClass:
+  # Must be set to enable snapshotting tests
+  FromName: true
```

Comment:
need to set to false?

Reply:
I don't think that makes a difference, testing snapshots is disabled atm